app/helpers/arclight_helper.rb

```diff
@@ -34,7 +34,7 @@ def collection_count
   end
 
   def repositories_active?
-    controller_name == 'repositories'
+    controller_name == 'repositories' && action_name == 'index'
```

Comment:
so i think the behavior is correct but the page is missing a breadcrumb which i think is the source of the confusion here.